media/stylus/components/wiki/page-buttons.styl

```diff
@@ -34,6 +34,15 @@
     }
 }
 
+@media $media-query-small-desktop {
+
+    .page-buttons, #page-buttons {
+        &.fixed {
+            position: relative;
+        }
+    }
+}
+
```

Comment:
Since we already have a couple other media queries separated out I think where it is is the best place for it.